src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedControllerTest.java

```diff
@@ -41,6 +42,9 @@
     @MockBean
     private ConsentOrderApprovedDocumentService service;
 
+    @Value("${feature.approved-consent-order-notification-letter}")
```

Comment:
shouldn't we been mocking this value out in test files rather than using the toggle value?

Reply:
would need to know more how this mocking would work - with toggle tests will work with toggle on and off, but maybe I'm missing something